libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1010,7 +1080,10 @@ void IRGeneratorForStatements::setLValue(Expression const& _expression, unique_p
 		// Do not define the expression, so it cannot be used as value.
 		m_currentLValue = std::move(_lvalue);
 	else
-		defineExpression(_expression) << _lvalue->retrieveValue() << "\n";
+	{
```

Comment:
Is this a leftover intermediate refactoring?

Reply:
No, this is making sure that we're not appending while calling `retrieveValue()`

Reply:
Ah right, yes!